pandas/tests/arrays/test_numpy.py

```diff
@@ -0,0 +1,192 @@
+"""
+Additional tests for PandasArray that aren't covered by
+the interface tests.
+"""
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+import pandas as pd
+from pandas.arrays import PandasArray
+from pandas.core.arrays.numpy_ import PandasDtype
+import pandas.util.testing as tm
+
+
+@pytest.fixture(params=[
```

Comment:
It doesn't make sense to reuse `any_numpy_dtype` or combine this with L37 below because they have different values for the `expected`.